tests/unit/test_zowe_core.py

```diff
@@ -258,6 +258,37 @@ def test_custom_file_and_custom_profile_loading(self, get_pass_func):
         }
         self.assertEqual(props, expected_props)
 
+    @patch("keyring.get_password", side_effect=keyring_get_password)
+    def test_custom_file_and_custom_profile_loading(self, get_pass_func):
+        """
+        Test loading of correct file given a filename and directory,
+        also load by profile_name correctly populating fields from custom
+        profile and secure credentials
+        """
+        # Setup - copy profile to fake filesystem created by pyfakefs
+        custom_file_path = os.path.join(self.custom_dir, self.custom_filename)
+        shutil.copy(self.original_file_path, custom_file_path)
+
+        self.setUpCreds(custom_file_path, {
+            "profiles.zosmf.properties.user": "user",
+            "profiles.zosmf.properties.password": "password",
+        })
+
+        # Test
+        prof_manager = ProfileManager(appname=self.custom_appname)
+        prof_manager.config_dir = self.custom_dir
+        props: dict = prof_manager.load(profile_name="lpar1.zosmf")
```

Comment:
It is giving error in this line. Any suggestions?

Reply:
@aadityasinha-dotcom I approved the workflow and it seems that now this line is working but there are 2 tests failing:
```
FAILED tests/unit/test_zowe_core.py::TestZosmfProfileManager::test_custom_file_and_custom_profile_loading
FAILED tests/unit/test_zowe_core.py::TestZosmfProfileManager::test_profile_loading_with_user_overriden_properties
```

Could you please investigate these failing tests? Happy to help if you have any questions 🙂